src/elements/content-sidebar/stories/tests/BoxAISidebar-visual.stories.tsx

```diff
@@ -24,3 +24,12 @@ export const BoxAIInSidebar: StoryObj<typeof BoxAISidebar> = {
         expect(sidebar).toBeInTheDocument();
     },
 };
+
+export const ExpandButtonCheck: StoryObj<typeof BoxAISidebar> = {
+    play: async ({ canvasElement }) => {
+        const canvas = within(canvasElement);
+
+        const expandButton = await canvas.findByRole('button', { name: 'Expand' }, { timeout: 5000 });
+        await userEvent.click(expandButton);
```

Comment:
At the moment the this tests only checks if button can be clicked. But not if anything happens. I think we should add at least an `expect` if callback function was called.